### ISSUE TITLE
Transportation processing error

### ISSUE THREAD

**Issue by pnorman** [pnorman/ClearTables#85]
Failed to execute lua function for basic tag processing: transportation.lua:80: attempt to perform arithmetic on a nil value

**Comment by pnorman** [comment]
ways are ~5089k, ~55105k in planet file,

lua is
```lua
if string.find(v, "^(%d+.?%d*) ?mph$") then
    return tostring(tonumber(string.match(v, "^(%d+.?%d*) ?mph$"))*1.609)
end
```

**Comment by pnorman** [comment]
Fixed in ff69d09
